### PR TITLE
[CHK-2575] build(deps): update commons

### PR DESCRIPTION
**Pull request by giovanniberti**
#### List of Changes

 * update commons version: remap brand on Redirect payments to "N/A" as `authorizationRequestedData.paymentMethodType` field was removed

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Depends on pagopa/pagopa-ecommerce-commons#184

### CODE REVIEW

File: src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt [hunk 1]
```diff
@@ -221,8 +221,7 @@ fun getBrand(authorizationRequestedData: TransactionGatewayAuthorizationRequeste
         is NpgTransactionGatewayAuthorizationRequestedData -> authorizationRequestedData.brand
         is PgsTransactionGatewayAuthorizationRequestedData ->
             authorizationRequestedData.brand?.toString()
-        is RedirectTransactionGatewayAuthorizationRequestedData ->
-            authorizationRequestedData.paymentMethodType.toString()
+        is RedirectTransactionGatewayAuthorizationRequestedData -> "N/A"
```
Comment (pietro-tota) [inline src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt:224]: what about put here the payment type code? 
But maybe it should be a different field in the response 🤔

**Review by pietro-tota — APPROVED** [review]

File: build.gradle.kts [hunk 2]
```diff
@@ -20,7 +20,7 @@ plugins {
   application
 }
 // eCommerce commons library version
-val ecommerceCommonsVersion = "1.10.1"
+val ecommerceCommonsVersion = "1.10.2"
```
Comment (pietro-tota) [inline build.gradle.kts:23]: update gradle.lockfile too

File: src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt (hunk 1 above)
Comment (giovanniberti) [reply src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt:224]: If we do need it I would address it in another PR.
For now we do have the `paymentMethodName` which is enough to get which redirect gateway was chosen

File: build.gradle.kts (hunk 2 above)
Comment (giovanniberti) [reply build.gradle.kts:23]: Fixed in [fd1376d](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/pull/90/commits/fd1376d470414748aea5b3b40bffd880337fc297)

**Pull request merged**